liquidprompt

```diff
@@ -1183,6 +1185,31 @@ _lp_ruby_env_color() {
     lp_ruby_env_color="[${LP_COLOR_VIRTUALENV}${lp_ruby_env}${NO_COL}]"
 }
 
+_lp_terraform_env() {
+    (( LP_ENABLE_TERRAFORM )) || return 2
+
+    local ret
+
+    if [ -d .terraform ]; then
+        local _tf_workspace
+        _tf_workspace="$(command terraform workspace show 2>/dev/null)"
```

Comment:
The preferred way to avoid calling an alias is using the `\terraform` syntax instead of `command terraform`. They both avoid aliases, but the backslash form allows for functions, which are useful for both users and mocking tests.